packages/manager/modules/billing/src/components/split-payment/action/controller.js

```diff
@@ -0,0 +1,28 @@
+export default class BillingPaymentMethodSplitPaymentActionCtrl {
+  /* @ngInject */
+  constructor($translate) {
+    this.$translate = $translate;
+  }
+
+  makeAction() {
```

Comment:
Not really verbose, is it intended ?

Reply:
The action can be to `activate` or to `deactivate`, will change the name